k8s/crds/kops.k8s.io_clusters.yaml

```diff
@@ -5369,6 +5369,19 @@ spec:
                 description: Configuration of subnets we are targeting
                 items:
                   properties:
+                    additionalRoutes:
+                      description: AdditionalRoutes to attach to the subnet's route
+                        table
+                      items:
+                        properties:
+                          cidr:
+                            description: CIDR destination of the route
+                            type: string
+                          target:
```

Comment:
Nothing stands out in this PR, but we will want to keep in mind future support for prefix lists instead of CIDRs for routes. We already support them in security group rules (https://github.com/kubernetes/kops/pull/13113). Essentially it would make the `cidr` field optional if a prefix list field is set.

Reply:
If you think support prefix list is important for the future I can add it to this PR.
Maybe we can replace `CIDR` with `Destination` instead of having two fields:

```yaml
additionalRoutes:
  description: AdditionalRoutes to attach to the subnet's route table
  items:
    properties:
      destination:
        description: CIDR destination or prefix list of the route
        type: string
      target:
        description: Target of the route
        type: string
      type: object
    type: array
```
Then the validation check if it is a valid CIDR or prefix list